test/unit/definitions/divorce/CaseEventToFields.test.js

```diff
@@ -3,7 +3,7 @@ const { expect, assert } = require('chai');
 const { find } = require('lodash');
 
 const caseEvent = Object.assign(require('definitions/divorce/json/CaseEvent'), []);
-const caseField = Object.assign(require('definitions/divorce/json/CaseField/CaseField'), []);
+const caseField = [].concat(require('definitions/divorce/json/CaseField/CaseField.json')).concat(require('definitions/divorce/json/CaseField/CaseField-prod.json'));
```

Comment:
I suppose we should test here both options - non-prod and prod. 2 test cases.

Reply:
In all the other test scenarios, we're only testing prod, right?

Reply:
I'm not so sure :)

```
describe('AuthorisationCaseEvent', () => {
  it('should contain a unique case type, case event ID and role (no duplicates) for non-prod', () => {
    const nonProd = mergeJsonFilesFor('nonprod');
    const uniqResult = uniqWith(nonProd, isDuplicated);

    expect(uniqResult).to.eql(nonProd);
  });

  it('should contain a unique case type, case event ID and role (no duplicates) for prod only', () => {
    const prodOnly = mergeJsonFilesFor('prod');
    const uniqResult = uniqWith(prodOnly, isDuplicated);

    expect(uniqResult).to.eql(prodOnly);
  });
});
```

div-ccd-definitions/test/unit/definitions/divorce/AuthorisationCaseEvent.test.js